rosidl_typesupport_introspection_c/cmake/rosidl_typesupport_introspection_c_generate_interfaces.cmake

```diff
@@ -12,6 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+if(NOT DEFINED _ROSIDL_TYPESUPPORT_INTROSPECTION_C_GENERATE_INTERFACES)
+  set(_ROSIDL_TYPESUPPORT_INTROSPECTION_C_GENERATE_INTERFACES TRUE)
+else()
+  return()
+endif()
+
+# Make sure that `rosidl_generator_c_generate_interfaces.cmake` extension point is executed first.
+include("${rosidl_generator_c_DIR}/rosidl_generator_c_generate_interfaces.cmake")
```

Comment:
This approach bypassing the extension point registration seems wrong to me. Imo this should be handled on the `find_package` level instead. A generator A which needs generator B to be run before should make sure to find_package it *before* it registers its own extension.

Reply:
Done